src/pkg/projects/netcoreapp/src/netcoreapp.depproj

```diff
@@ -60,7 +60,14 @@
       <_ToolsToPackage Include="$(_runtimePackageDir)tools/**/*.*"/>
       <FilesToPackage Include="@(_ToolsToPackage)">
         <NuGetPackageId>$(_runtimePackageId)</NuGetPackageId>
-        <TargetPath>tools/%(RecursiveDir)</TargetPath>
+        <!--
+          Trim '/' at the end of the target path, if exists. The packaging tools automatically add
```

Comment:
We should fix that in packaging tools.

Reply:
True, filed https://github.com/dotnet/arcade/issues/2998.